### PR TITLE
Bump promises dependency

### PR DESCRIPTION
**Pull request by ncooke3**

### CODE REVIEW

**Review by paulb777 — COMMENTED** [review]
It looks like Package.swift is missing Promises completely. The dependency list should match between the two package managers and shouldn't rely on transitive deps.

**Comment by ncooke3** [comment]
Was able to manually verify CocoaPods. Was also able to verify SPM after pointing the manifest's GUL's dependency to the latest revision.

**Review by paulb777 — COMMENTED** [review]
Add changelog and podspec version update

File: GoogleDataTransport.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'GoogleDataTransport'
-  s.version          = '9.0.1'
```
Comment (paulb777) [inline GoogleDataTransport.podspec:3]: 9.1.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# v9.0.2
```
Comment (paulb777) [inline CHANGELOG.md:1]: 9.1.0 here too

Comment (ncooke3) [reply CHANGELOG.md:1]: Now I'm just getting sloppy! 🤦

Comment (paulb777) [reply CHANGELOG.md]: Lot's of details! 😄

**Review by paulb777 — APPROVED** [review]

**Pull request merged**